network-proto/node.proto

```diff
@@ -89,6 +89,18 @@ message RecordTransactionResponse {
     bytes id = 2;
 }
 
+
+// Gossip message.
+message GossipMessage {
+    message NodeId {
+      fixed64 high = 1;
+      fixed64 low = 2;
```

Comment:
Why do we want to specify `NodeId` in this way, and not with `bytes` of an expected size? Is there any special meaning for the 64-bit halves?

Reply:
I get this idea, but I don't think how we can preserve the current properties we want without major code complication :(

Reply:
What are the properties, other than the bit size of the `NodeId`?